packages/fast-tooling-react/src/css-editor/editor.tsx

```diff
@@ -1,24 +1,16 @@
 import React from "react";
-import { get, omit, pick } from "lodash-es";
-import Foundation, {
-    FoundationProps,
-    HandledProps,
-} from "@microsoft/fast-components-foundation-react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
 import {
-    CSSComponent,
     CSSEditorHandledProps,
+    CSSEditorProps,
     CSSEditorUnhandledProps,
-    CSSEditorValues,
 } from "./editor.props";
-import { CSSPosition, CSSPositionValues, Location, PositionValue } from "./position";
-import { CSSSpacing, CSSSpacingValues, SpacingProperty } from "./spacing";
-import { CSSWidth, CSSWidthValues } from "./width";
-import { CSSHeight, CSSHeightValues } from "./height";
-import { CSSColor, CSSColorValues } from "./color";
-import { CSSBorder, CSSBorderValues } from "./border";
-import { CSSBoxShadow, CSSBoxShadowValues } from "./box-shadow";
-import { CSSBackground, CSSBackgroundValues } from "./background";
-import { CSSBorderRadius, CSSBorderRadiusValues } from "./border-radius";
+import cssEditorDataSchema from "./editor-data.schema";
+import { CSSColor } from "./color";
+import { getDataFromSchema } from "../../src/data-utilities";
```

Comment:
there is no src folder, use pathing relative to the internals of the src folder or this will fail when imported